ui/components/multichain/app-footer/app-footer.js

```diff
@@ -55,7 +55,10 @@ export const AppFooter = () => {
   const activeConnections = location.pathname === CONNECTIONS;
   const isUnlocked = useSelector((state) => state.metamask.isUnlocked);
   const isFullScreen = getEnvironmentType() === ENVIRONMENT_TYPE_FULLSCREEN;
-  const selectedAddress = useSelector(getSelectedAddress);
+  const selectedAccount = useSelector(getSelectedInternalAccount);
+
+  // During onboarding there is no selected internal account
+  const selectedAddress = selectedAccount?.address;
```

Comment:
Would `const selectedAddress = useSelector(getSelectedAddress)` ever return null? Would a undefined value have an effect on 
```ts
  const connectedSite = connectedSites[selectedAddress]?.find(
    ({ origin }) => origin === currentTabOrigin,
  );
  ```

Reply:
It could during onboarding when there aren't any accounts. The connected sites wouldn't be found, but thats ok since we are still setting up the account.